app/assets/stylesheets/application.css.scss

```diff
@@ -39,3 +39,9 @@
     border: solid 1px $dark-red;
   }
 }
+
+.pagination {
+  text-align: center;
+  background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.